src/Gui/OverlayWidgets.cpp

```diff
@@ -1152,7 +1148,6 @@ void OverlayTabWidget::setTransparent(bool enable)
         return;
     if(hGrp) {
         Base::StateLocker lock(_saving);
-        hGrp->SetBool("Transparent", enable);

```

Comment:
Small mistake here. Remove the line containing `setProperty("transparent", enable);` which is a couple of lines higher and add this line back.

Reply:
done